content/utilities/player-list/plugin.js

```diff
@@ -172,6 +172,23 @@ function Plugin() {
 			for (var group of groups) {
 				var th = document.createElement('th');
```

Comment:
this can be changed from var to let
`let th = document.createElement('th');`
then the anonymous function below can be deleted (the code inside put outside)
```
(function(th){
})(th);
```
I should have always used let for variables